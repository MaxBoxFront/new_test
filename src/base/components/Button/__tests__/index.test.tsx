import React from 'react';
import renderer from 'react-test-renderer';

import { Button } from 'base/components/Button';

describe('The Button component', () => {
	it('should renders correctly', () => {
		const tsx = (<Button />);
		const tree = renderer
			.create(tsx)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
