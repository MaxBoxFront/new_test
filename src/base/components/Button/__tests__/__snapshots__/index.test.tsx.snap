// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Button component should renders correctly 1`] = `
<button
  className="button"
  type="button"
/>
`;
